src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonMoveBookingIntTest.kt

```diff
@@ -383,71 +644,91 @@ fun nomisPhysicalAttributesToOffenderRemeasured(
   bookingId: Long,
   bookingStartTime: LocalDateTime = LocalDateTime.now().minusDays(1),
 ) =
-  PrisonerPhysicalAttributesResponse(
+  prisonerPhysicalAttributes(
     offenderNo = offenderNo,
     bookings = listOf(
       // The to offender's old booking
-      BookingPhysicalAttributesResponse(
+      bookingPhysicalAttributes(
         bookingId = 3456L,
-        startDateTime = "${bookingStartTime.minusDays(8)}",
-        endDateTime = "${bookingStartTime.minusDays(6)}",
+        activeBooking = false,
         latestBooking = false,
         physicalAttributes = listOf(
-          PhysicalAttributesResponse(
-            attributeSequence = 1,
-            heightCentimetres = 170,
-            weightKilograms = 70,
-            createdBy = "A_USER",
-            createDateTime = "${bookingStartTime.minusDays(7)}",
-            modifiedBy = "ANOTHER_USER",
+          physicalAttributes(
+            height = 170,
+            weight = 70,
+            createDateTime = bookingStartTime.minusDays(7),
             modifiedDateTime = null,
-            auditModuleName = "MODULE",
           ),
         ),
       ),
       // The to offender's new booking has weight re-entered hence new weight and modified time after booking start time
-      BookingPhysicalAttributesResponse(
+      bookingPhysicalAttributes(
         bookingId = bookingId,
-        startDateTime = "$bookingStartTime",
-        endDateTime = null,
+        activeBooking = true,
         latestBooking = true,
         physicalAttributes = listOf(
-          PhysicalAttributesResponse(
-            attributeSequence = 1,
-            heightCentimetres = 170,
-            weightKilograms = 90,
-            createdBy = "A_USER",
-            createDateTime = "${bookingStartTime.minusDays(7)}",
-            modifiedBy = "ANOTHER_USER",
-            modifiedDateTime = "${bookingStartTime.plusDays(1)}",
-            auditModuleName = "MODULE",
+          physicalAttributes(
+            height = 170,
+            weight = 90,
+            createDateTime = bookingStartTime.minusDays(7),
+            modifiedDateTime = bookingStartTime.plusDays(1),
           ),
         ),
       ),
     ),
   )
 
+fun physicalAttributes(
+  height: Int,
+  weight: Int,
+  createDateTime: LocalDateTime,
+  modifiedDateTime: LocalDateTime?,
+) = PhysicalAttributesResponse(
```

Comment:
Tried to improve the test data builders but it's still a lot of code because they are quite complex.